src/cli/README.md

```diff
@@ -367,19 +367,61 @@ Done
 Done
 ```
 
-### br omrprefix
+### br info
 
-Get the randomly generated off-mesh-routable prefix of the Border Router.
+Print info about the border routing manager.
 
 ```bash
-> br omrprefix
+> br info
+State: running
+OMR prefix
+    Local: fdc1:4e10:2ef7:1::/64
+    Favored: fdc1:4e10:2ef7:1::/64 prf:med
+On-link prefix
+    Local: fdde:ad00:beef:cafe::/64
+    Favored: 1200:abba:baba:0::/64
+Discovered Prefix Table
+    prefix:fd00:1234:5678:0::/64, on-link:no, ms-since-rx:29526, lifetime:1800, route-prf:med, router:ff02:0:0:0:0:0:0:1
+    prefix:1200:abba:baba:0::/64, on-link:yes, ms-since-rx:29527, lifetime:1800, preferred:1800, router:ff02:0:0:0:0:0:0:1
+Counters
+    Inbound Unicast: Packets 4 Bytes 320
+    Inbound Multicast: Packets 0 Bytes 0
+    Outbound Unicast: Packets 2 Bytes 160
+    Outbound Multicast: Packets 0 Bytes 0
+    RA Rx: 4
+    RA TxSuccess: 2
+    RA TxFailed: 0
+    RS Rx: 0
+    RS TxSuccess: 2
+    RS TxFailed: 0
+Done
+```
+
+### br favorednat64prefix
+
+Get the favored NAT64 prefix of the Border Router.
+
+`OPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE` is required.
+
+```bash
+> br favorednat64prefix
```

Comment:
Should we have a common prefix and specify local vs. favored using another arg? For example:
```
br nat64prefix favored
```

Reply:
Good idea. 
We can have `br omrprefix` on its own to output both "local" and "favored" and then allow specific ones with the second arg.